tools/cmdline.cc

```diff
@@ -53,9 +53,30 @@ bool CommandLineParser::Parse(int argc, const char* argv[]) {
       i++;
       continue;
     }
+    if (!strcmp("-hv", argv[i]) || !strcmp("-vh", argv[i])) {
```

Comment:
I'd probably leave this out of this PR. I also think it is possible to implement it such you can bundle any once char flags rather than checking for these specific combinations.

Reply:
It seems like `v` and `h` are handled separately from the other short flags in this section of code.  So bundling short options probably won't affect `v` and `h`.  Should this be in a separate pull request?

Reply:
yeah, let's do this in another PR and discuss the distance range here instead.

Reply:
Okay.  I'll push an update to remove this.

Reply:
I would also expect it to handle options like `-me9`, which would be modular encoding with effort 9, similar to how most GNU programs work. Maybe even `-e9d2` for effort 9 and distance 2, although I don't know if any programs support this. `du -d1t20` doesn't work, but `du -d1 -t20` does.

Reply:
Maybe I should open it as a feature request rather than a pull request.